src/codestyle.sh

```diff
@@ -84,6 +127,10 @@ function codestyle_main()
     cmd_manager "$flag" "$cmd_script $file"
     [[ "$?" != 0 ]] && say "$SEPARATOR"
   done
+
+  if [[ -n "$range_option_flag" ]]; then
+    rm "$temp_file"
```

Comment:
Never use `rm` with a bash variable without checking if the variable has the correct value. Check `is_safe_path_to_remove`.